src/pages/Signup/UserInfo.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+const UserInfo = () => {
+  const navigate = useNavigate();
+
+  const userData = () => {
+    fetch(`http://10.58.5.43:8000/users/additional-info`, {
+      method: 'PATCH',
+      headers: { Authorization: localStorage.getItem('Token') },
+      body: JSON.stringify({
+        last_name: LastName,
+        first_name: FirstName,
+        phone_number: PhoneNumber,
+        birth_day: BirthDay,
+      }),
+    })
+      .then(response => response.json())
+      .then(data => {
+        if (data.message === 'USER_INFO_UPDATED') {
+          alert('회원가입에 성공했습니다!');
+          navigate('/main');
+        } else if (data.message === 'KEY_ERROR') {
+          alert(MESSAAGE_ALERT.KEY_ERROR);
+        } else if (data.message === 'FISRT_NAME_ERROR') {
+          alert(MESSAAGE_ALERT.FISRT_NAME_ERROR);
+        } else if (data.message === 'LAST_NAME_ERRO') {
+          alert(MESSAAGE_ALERT.LAST_NAME_ERRO);
+        } else if (data.message === 'PHONE_NUMBER_ERROR') {
+          alert(MESSAAGE_ALERT.PHONE_NUMBER_ERROR);
+        } else if (data.message === 'PHONE_NUMBER_ALREADY_EXIST') {
+          alert(MESSAAGE_ALERT.PHONE_NUMVER_ALREADY_EXIST);
+        }
```

Comment:
```suggestion
        } else {
          alert(`${MESSAAGE_ALERT[data.message]}`);
        }
```
상수데이터는 잘 만들어 주셨는데,
그 부분을 이렇게 활용하면 더 간략하게 표현이 가능합니다!
혹시 이해가 안 되신다면 라운지 방문해주세요!

Reply:
수정했습니다:)